src/platform/plugins/shared/unified_doc_viewer/public/components/content_framework/section/section.tsx

```diff
@@ -62,39 +63,43 @@ export function ContentFrameworkSection({
   hasPadding = true,
 }: ContentFrameworkSectionProps) {
   const [accordionState, setAccordionState] = useState<EuiAccordionProps['forceState']>(forceState);
+  const renderActions = () => {
+    const size = 'xs';
+    return (
+      <>
+        <EuiFlexGroup gutterSize="s" justifyContent="flexEnd" alignItems="center">
+          {actions?.map((action, idx) => {
+            const { icon, ariaLabel, dataTestSubj, label, onClick, href } = action;
```

Comment:
Should we move the actions sanity check to about rendering unnecessary component?

```suggestion
{actions ? 
        <EuiFlexGroup gutterSize="s" justifyContent="flexEnd" alignItems="center">
          {actions.map((action, idx) => {
            const { icon, ariaLabel, dataTestSubj, label, onClick, href } = action;
```

Reply:
We already check `actions?.length` before rendering `renderActions`.

Anyway, I tweaked the logic a bit: we now have `SectionActions` in a separate file, which makes the component cleaner.

https://github.com/elastic/kibana/pull/236567/commits/8b5083885b990aa1bc03df01a8c65080b5ee4779